actions/st2_prep_release_for_st2_pkg.sh

```diff
@@ -101,8 +101,9 @@ fi
 # Update the st2 version at circle.yml
 CIRCLE_YML_FILE="circle.yml"
 echo "Setting version in ${CIRCLE_YML_FILE} to ${BRANCH}..."
-sed -i -e "s/\(ST2_GITREV:[ ]*\).*/\1${BRANCH}/" ${CIRCLE_YML_FILE}
-sed -i -e "s/\(ST2MISTRAL_GITREV:[ ]*\).*/\1${MISTRAL_VERSION}/" ${CIRCLE_YML_FILE}
+# capture group one is to ensure proper indentation. Capture group two is actual var definition
+sed -i -e "s/\(.*\)# \(ST2_GITREV:[ ]*\).*/\1\2${BRANCH}/" ${CIRCLE_YML_FILE}
```

Comment:
Okay, updated in https://github.com/StackStorm/st2cd/pull/249/commits/37435cde7c583f6f6d7b6e809775c4f5bf9cc57c